src/main/java/de/tum/in/www1/artemis/service/connectors/LocalVCUserManagementService.java

```diff
@@ -0,0 +1,102 @@
+package de.tum.in.www1.artemis.service.connectors;
+
+import java.util.Set;
+
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Service;
+
+import de.tum.in.www1.artemis.domain.Course;
+import de.tum.in.www1.artemis.domain.User;
+import de.tum.in.www1.artemis.exception.VersionControlException;
+
+@Service
+@Profile("localvc")
+public class LocalVCUserManagementService implements VcsUserManagementService {
+
+    public LocalVCUserManagementService() {
+    }
+
+    /**
+     * Creates a new user in the VCS based on a local Artemis user. Should be called
+     * if Artemis handles user creation
+     * and management
+     *
+     * @param user     The local Artemis user, which will be available in the VCS
+     *                 after invoking this method
+     * @param password the password of the user to be set
+     */
+    @Override
+    public void createVcsUser(User user, String password) throws VersionControlException {
+        // Not implemented for local VC.
```

Comment:
```suggestion
        // Not implemented for local VC.
        throw new UnsupportedOperaationException();
```

Or should these methods do nothing?